components/Image/image.css

```diff
@@ -0,0 +1,35 @@
+.images-wrapper {
+	align-items: center;
+	align-self: center;
+	background-color: var(--white);
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	justify-self: center;
+	margin: 2rem;
+	min-width: 20rem;
+	padding: 2rem;
+	width: 80%;
+}
+.img-container {
+	height: 15rem;
+	width: 15rem;
+}
+.img-round,
+.img-rounded-corner,
+.img-responsive {
+	height: 100%;
+	width: 100%;
+}
+.img-container,
+.img-round,
+.img-rounded-corner,
+.img-responsive {
+	object-fit: cover;
+}
```

Comment:
This code is awesome.
But a small suggestion would be to have a common *img* class having common properties.

Reply:
Done